README.md

```diff
@@ -83,443 +84,446 @@ Thus, "Jun+Wei" represents the names "Jun-Wei", "Jun Wei" and "Junwei".
-Equivalent name pairs   :     734
-
-Names from Great Britain  :  2232
-Names from Ireland        :   876
-Names from U.S.A.         :  3916
-Names from Italy          :  2871
-Names from Malta          :   526
-Names from Portugal       :  1041
-Names from Spain          :  1645
-Names from France         :  1821
-Names from Belgium        :  1518
-Names from Luxembourg     :   564
-Names from the Netherlands:  3302
-Names from East Frisia    :  2589   ("Ostfriesland")
-Names from Germany        :  2636
-Names from Austria        :  1753
-Names from Swiss          :  2464
-Names from Iceland        :  1302
-Names from Denmark        :  1139
-Names from Norway         :  1064
-Names from Sweden         :   974
-Names from Finland        :   873
-Names from Estonia        :  1121
-Names from Latvia         :   710
-Names from Lithuania      :   845
-Names from Poland         :   359
-Names from Czech Republic :   477
-Names from Slovakia       :   477
-Names from Hungary        :   371
-Names from Romania        :  1971
-Names from Bulgaria       :  1793
-Names from Bosnia and Herzegovina:  1216
-Names from Croatia        :   894
-Names from Kosovo         :  1002
-Names from Macedonia      :   851
-Names from Montenegro     :   621
-Names from Serbia         :   727
-Names from Slovenia       :   636
-Names from Albania        :  1748
-Names from Greece         :   778
-Names from Russia         :   489
-Names from Belarus        :   508
-Names from Moldova        :   405
-Names from Ukraine        :   531
-Names from Armenia        :   653
-Names from Azerbaijan     :   764
-Names from Georgia        :   293
-Names from Kazakhstan/Uzbekistan,etc.:   862
-Names from Turkey         :  1806
-Names from Arabia/Persia  :  2025
-Names from Israel         :  1067
-Names from China          :  7334
-Names from India/Sri Lanka:  1455
-Names from Japan          :  1384
-Names from Korea          :  1376
-Names from Vietnam        :   307
+| category              |  number |
+| --------              |  ------ |
+| Number of first names |   44568 |
+| – girl's names        |   16921 |
+| – boy's  names        |   17740 |
+| – unisex names        |    9907 |
+| Equivalent name pairs |     734 |
+
+| area                                   | number |
+| ----                                   | ------ |
+| Names from Great Britain               |  2232  |
+| Names from Ireland                     |   876  |
+| Names from U.S.A.                      |  3916  |
+| Names from Italy                       |  2871  |
+| Names from Malta                       |   526  |
+| Names from Portugal                    |  1041  |
+| Names from Spain                       |  1645  |
+| Names from France                      |  1821  |
+| Names from Belgium                     |  1518  |
+| Names from Luxembourg                  |   564  |
+| Names from the Netherlands             |  3302  |
+| Names from East Frisia                 |  2589 ("Ostfriesland") |
```

Comment:
so the alignment looks nice, I suggest to move the bracket term to a third column.

Reply:
I made a third column.